docs/first_pr.md

```diff
@@ -0,0 +1,61 @@
+# Getting started with EvalAI Issues:
+
+To contribute for CloudCV in EvalAI is a facile task. Please follow these steps in order to get started :-)
+
+**Step 1 : Fork**
+
+1. Fork the EvalAI repository from [this](https://github.com/Cloud-CV/EvalAI) link.
+
+**Step 2 : Picking up a issue:**
+
+1. Pick up a suitable issue that you find will be facile for you to fix. Moreover, you can also 
+take the issues based on their labels. Almost every issue is marked with a label according to its difficulty.
+
+2. After selecting any issue, mention any mentor’s name there and ask him to assign it to you.
+
+3. Once it gets assigned , create a branch from your fork’s updated master branch using the command :-
+`git checkout -b branch_name`
+**Note:** To know more about the `checkout` command command, please refer to [this](https://git-scm.com/docs/git-checkout) link.
+
+4. Start working on the issue.
+
+**Step 3: Committing Your Changes:**
+
+1. After making the required feature as asked in the issue, you need to add your files to local git repository.
+
+2. If you want to create any migration file then use this command to create the migration file.
```

Comment:
Here we are writing steps for how to commit changes so this should not be a part of this document. It should be addressed in a separate document.